sql/src/main/java/io/crate/operation/tablefunctions/BlankTableFunction.java

```diff
@@ -0,0 +1,143 @@
+import io.crate.metadata.Signature;
+import io.crate.metadata.TableIdent;
+import io.crate.metadata.table.StaticTableInfo;
+import io.crate.metadata.table.TableInfo;
+import io.crate.metadata.tablefunctions.TableFunctionImplementation;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import org.elasticsearch.cluster.service.ClusterService;
+
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+/**
+ * Generates a one row, no column, empty table.
+ */
+public class BlankTableFunction {
+
+    private final static String NAME = "blank";
+    private final static TableIdent TABLE_IDENT = new TableIdent(null, NAME);
+
+    static class BlankTableFunctionImplementation implements TableFunctionImplementation {
+
+        private final FunctionInfo info;
+
+        private BlankTableFunctionImplementation(FunctionInfo info) {
+            this.info = info;
+        }
+
+        @Override
+        public FunctionInfo info() {
+            return info;
+        }
+
+        @Override
+        public Bucket execute(Collection<? extends Input> arguments) {
+            return new Bucket() {
+                final Object[] cells = new Object[0];
+                final RowN row = new RowN(cells);
+
+                @Override
+                public int size() {
+                    return 1;
+                }
+
+                @Override
+                public Iterator<Row> iterator() {
+                    return new Iterator<Row>() {
+
+                        int currentRow = 0;
+
+                        @Override
+                        public boolean hasNext() {
+                            return currentRow < 1;
+                        }
+
+                        @Override
+                        public Row next() {
+                            if (!hasNext()) {
+                                throw new NoSuchElementException("No more rows");
+                            }
+                            currentRow++;
+                            return row;
+                        }
+
+                        @Override
+                        public void remove() {
+                            throw new UnsupportedOperationException("remove is not supported for " +
+                                                                    BlankTableFunction.class.getSimpleName() +
+                                                                    "$iterator");
+                        }
+                    };
+                }
+            };
+        }
+
+        @Override
+        public TableInfo createTableInfo(ClusterService clusterService) {
+            final String localNodeId = clusterService.localNode().getId();
+            return new StaticTableInfo(TABLE_IDENT, new LinkedHashMap<>(0), null, Collections.emptyList()) {
```

Comment:
I think there is no need for an LinkedHashMap, wouldn't Collections.emptyMap() work?